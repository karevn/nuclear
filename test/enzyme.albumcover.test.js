import React from 'react';
import { shallow } from 'enzyme';

import AlbumCover from '../app/components/AlbumCover';

describe('<AlbumCover />', () => {
  it('renders an album cover', () => {
    const wrapper = shallow(
      <AlbumCover />
    );
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.containsMatchingElement(
      <div className='album_cover_container' />
    ));
  });
});
