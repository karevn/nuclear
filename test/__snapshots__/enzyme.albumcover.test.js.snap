// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlbumCover /> renders an album cover 1`] = `
<div
  className="album_cover_container"
  style={Object {}}
>
  <AlbumOverlay />
  <img />
  <AlbumInfo />
</div>
`;
